spec/unit/actions/service_credential_binding_key_create_spec.rb

```diff
@@ -250,6 +250,34 @@ module V3
             end
           end
         end
+
+        context 'when creating keys with the same name concurrently' do
+          let(:service_instance2) { ManagedServiceInstance.make(space:) }
```

Comment:
We don't need a second service instance, name and message here. The constraint is on the name of the keys per service instance.